contents/guides/users/aspects.md

```diff
@@ -0,0 +1,41 @@
+---
+title: What are aspects and how do I use them?
+category: User guides
+---
+
+## What is an aspect?
+
+Think of your life and the people you know. Each person is a part of one or more aspects of your life. They might be a member of your family, a close friend, a work colleague or someone you play sport or music with or with whom you share a particular interest. They might be someone you just like sharing jokes with online. Or they might be part of more than one of these aspects of your life.
```

Comment:
End of second sentence: "who shares a particular interest with you" is less formal.

Reply:
yep, changed. :)